packages/perseus/src/widgets/radio/base-radio.jsx

```diff
@@ -599,4 +419,177 @@ class BaseRadio extends React.Component<$FlowFixMe, $FlowFixMe> {
     }
 }
 
+const styles: StyleDeclaration = StyleSheet.create({
+    // eslint-disable-next-line react-native/no-unused-styles
```

Comment:
**suggestion:** You should be able to remove all of these eslint-disable comments.